packages/orchestration/src/facade.js

```diff
@@ -168,14 +171,19 @@ export const makeOrchestrationFacade = ({
   chainHub,
   makeLocalChainAccountKit,
   makeRecorderKit,
+  asyncFlowTools,
 }) => {
-  console.log('makeOrchestrationFacade got', {
-    zone,
-    timerService,
-    zcf,
-    storageNode,
-    orchestrationService,
-  });
+  (zone &&
+    timerService &&
+    zcf &&
+    storageNode &&
+    orchestrationService &&
+    // @ts-expect-error type says defined but double check
+    makeLocalChainAccountKit &&
+    // @ts-expect-error type says defined but double check
+    makeRecorderKit &&
+    asyncFlowTools) ||
+    Fail`params missing`;
```

Comment:
Wasn't this already committed in https://github.com/Agoric/agoric-sdk/pull/9518 ?

Reply:
yeah, this was based on an earlier master